app/models/hackathon/brand.rb

```diff
@@ -2,7 +2,7 @@ module Hackathon::Brand
   extend ActiveSupport::Concern
 
   included do
-    validates :website, presence: true, format: URI::DEFAULT_PARSER.make_regexp(%w[http https])
+    validates :website, format: URI::DEFAULT_PARSER.make_regexp(%w[http https])
```

Comment:
Since we're validating the format, it'll still complain about a blank website. It's nice to be able to create a hackathon without a website (although it should be required on the create form).

There are two solutions here:

I'd prefer if we validate `on: :submit`
```suggestion
    validates :website, format: URI::DEFAULT_PARSER.make_regexp(%w[http https]), on: :submit
```

---

Alternatively, we could allow for blank:

```suggestion
    validates :website, format: URI::DEFAULT_PARSER.make_regexp(%w[http https]), allow_blank: true
```